### PR TITLE
[en] Update concepts/extend-kubernetes

### PR DESCRIPTION
**Pull request by ptux**
- put describtion of controller in the same paragraph
- keep consistency with subtitle (Extension Patterns)

### CODE REVIEW

**Comment by ptux** [comment]
@cheftako
@lavalamp

Could you guys be my sponsors to get the membership of Kubernetes?
Here are my contributions:

https://github.com/kubernetes/website/commits/main?author=ptux

**Comment by ptux** [comment]
I think I could review PRs for fixing typos, minor changes.
That's why I want to become a member. (:

**Comment by tengqm** [comment]
@ptux You don't have to be a member to contribute PRs or reviews.

File: content/en/docs/concepts/extend-kubernetes/_index.md [hunk 1]
```diff
@@ -68,14 +68,14 @@ clusters and managed installations.
...
 Kubernetes called the *Controller* pattern. Controllers typically read an
 object's `.spec`, possibly do things, and then update the object's `.status`.
+A controller is a client of Kubernetes.
 
-A controller is a client of Kubernetes. When Kubernetes is the client and
-calls out to a remote service, it is called a *Webhook*. The remote service
-is called a *Webhook Backend*. Like Controllers, Webhooks do add a point of
-failure.
+When Kubernetes is the client and calls out to a remote service, 
+it is called a *Webhook*. The remote serviceis called a *Webhook Backend*.
```
Comment (RinkiyaKeDad) [inline content/en/docs/concepts/extend-kubernetes/_index.md:74]:
```suggestion
it is called a *Webhook*. The remote service is called a *Webhook Backend*. 
```

File: content/en/docs/concepts/extend-kubernetes/_index.md [hunk 2]
```diff
@@ -68,14 +68,14 @@ clusters and managed installations.
...
 object's `.spec`, possibly do things, and then update the object's `.status`.
+A controller is a client of Kubernetes.
 
-A controller is a client of Kubernetes. When Kubernetes is the client and
-calls out to a remote service, it is called a *Webhook*. The remote service
-is called a *Webhook Backend*. Like Controllers, Webhooks do add a point of
-failure.
+When Kubernetes is the client and calls out to a remote service, 
+it is called a *Webhook*. The remote serviceis called a *Webhook Backend*. 
+Like Controllers, Webhooks do add a point of failure.
```
Comment (RinkiyaKeDad) [inline content/en/docs/concepts/extend-kubernetes/_index.md:75]: nit:
```suggestion
Like Controllers, Webhooks too add a point of failure.
```

File: content/en/docs/concepts/extend-kubernetes/_index.md [hunk 3]
```diff
@@ -68,14 +68,14 @@ clusters and managed installations.
...
 
-In the webhook model, Kubernetes makes a network request to a remote service.
-In the *Binary Plugin* model, Kubernetes executes a binary (program).
+In the *Webhook* pattern, Kubernetes makes a network request to a remote service.
+In the *Binary Plugin* pattern, Kubernetes executes a binary (program).
```
Comment (RinkiyaKeDad) [inline content/en/docs/concepts/extend-kubernetes/_index.md:78]: To me the use of "pattern" makes sense in "Controller pattern", but I think "model" fits better here. Let's see what other folks have to say :)

Comment (ptux) [reply content/en/docs/concepts/extend-kubernetes/_index.md:78]: Here is my reason for the modification.
> keep consistency with the subtitle (Extension Patterns)

Or we change the subtitle to `Extension Models`?

Comment (jimangel) [reply content/en/docs/concepts/extend-kubernetes/_index.md:78]: I think patterns can include models. Taking a look at the closest definition of each (from a search):

- **pattern**: a form or model proposed for imitation
- **model**: a system or thing used as an example to follow or imitate

In that context, I take pattern to be larger themes and model(s) to be within patterns.

**Comment by tengqm** [comment]
This PR makes no sense.
/close

**Pull request closed without merging**